packages/styled-components/package.json

```diff
@@ -89,13 +89,13 @@
     "eslint-plugin-react": "^7.12.4",
     "flow-bin": "^0.100.0",
     "prop-types": "^15.7.2",
-    "react": "^16.8.6",
+    "react": "^16.13.1",
```

Comment:
Can this not stay at `16.8.6`? I can't see any code that's depending on features past 16.8. 16.8.0 supports hooks

Reply:
I'll check again. Maybe I was wrong.

Reply:
You were right. Well done.